src/main/fc/fc_core.c

```diff
@@ -373,6 +374,13 @@ void tryArm(void)
         if (ARMING_FLAG(ARMED)) {
             return;
         }
+#if defined(USE_WATCHDOG)
```

Comment:
Even if `USE_WATCHDOG` is defined, shouldn't we make a setting to disable it programmatically via settings?